README.md

```diff
@@ -59,6 +59,42 @@ For details on the available methods we refer to [iic.jku.at/eda/research/quantu
 
 The simulator is based on the references listed below and can either be used as a **standalone executable** with command-line interface, or as a **library** for the incorporation in other projects.
 
+## Using the Python Bindings / Backend for Qiskit
+
+The backend for Qiskit is available via PyPi. The following code gives an example on the usage:
+
+```python
+from qiskit import *
+import jkq.ddsim
```

Comment:
maybe use `from jkq import ddsim`? then the `jkq` can be dropped in the following.